inc/css-builder.php

```diff
@@ -217,7 +217,9 @@ public function get_css() {
 				continue;
 			}
 
-			if ( $max_res < 1920 ) {
+			if ( $max_res == 0 && ! empty( $min_res ) ) {
+				$css_text .= '@media (min-width:' . intval( $min_res ) . 'px) {';
+			} elseif( $max_res < 1920 ) {
 				$css_text .= '@media (max-width:' . intval( $max_res ) . 'px)';
 				if( ! empty( $min_res ) ) {
```

Comment:
And one more `if ( ! empty( $min_res ) ) {`

Reply:
Can you please clarify what line you're suggesting that gets added to? I ask as it's checked on 220 and 223 so I'm not too sure what you're suggesting it needs to be added. Sorry about this.

Reply:
Line 224.